citi-onboarding/Eventos-CITi
# Feat/create footer

### What I did

Create a footer on client that hides itself on scroll

### How to test

1. Open the terminal and go to the folder of this project
2. In terminal type:

git checkout develop
git pull
git checkout feat/create-footer
cd client/
npm install
npm run start 

3. In your browser:
4.  Open localhost http://localhost:3000
5.  scroll the page down to the bottom and check if a footer shows up
6.  scroll up and check if the footer goes away
7.  change to mobile view by pressing F12 or ctrl+shift+I
8.  if necessary click on a phone icon on the top right of the page
9.  scroll to the bottom of the page and check if a footer shows up
10.  scroll up and check if the footer goes away
11.  scroll down again and check if a footer shows up
12.  tap in a arrow at the top of the footer and check if the footer goes away
